### PR TITLE
add optional is_kebechet metadata

### PR DESCRIPTION
**Pull request by KPostOffice**
Depends on https://github.com/thoth-station/common/pull/811, allows is_kebechet to be added to metadata.  If both origin and is_kebechet are present then we will trigger kebechet run-result in adviser workflow

### CODE REVIEW

File: thamos/lib.py [hunk 1]
```diff
@@ -325,6 +326,7 @@ def advise(
         "recommendation_type": recommendation_type,
         "debug": debug,
         "force": force,
+        "is_kebechet": metadata_is_kebechet,
```
Comment (pacospace) [inline thamos/lib.py:329]: why not simply `is_kebechet`? like for `is_s2i`

Comment (fridex) [reply thamos/lib.py:329]: you can directly pass the string representation of `enum`, following openapi on user-api

Comment (KPostOffice) [reply thamos/lib.py:329]: Is this implemented yet?

**Comment by pacospace** [comment]
Have a look here please: https://github.com/thoth-station/thamos/pull/421 :)

**Comment by KPostOffice** [comment]
@pacospace Noted I am closing this PR :+1:

**Pull request closed without merging**